lib/SILOptimizer/PassManager/PassManager.cpp

```diff
@@ -87,6 +87,9 @@ llvm::cl::list<std::string>
     SILDisablePass("sil-disable-pass", llvm::cl::CommaSeparated,
                      llvm::cl::desc("Disable passes "
                                     "which contain a string from this list"));
+llvm::cl::list<std::string> SILDisableOnlyFun(
+    "sil-disable-only-function", llvm::cl::CommaSeparated,
```

Comment:
Can you add pass to the name? Maybe: `-sil-disable-pass-only-function=`. Then it is really clear.